src/main/java/edu/kis/powp/jobs2d/drivers/adapter/SolidLineDrawerAdapter.java

```diff
@@ -0,0 +1,39 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import static edu.kis.powp.jobs2d.features.DrawerFeature.getDrawerController;
+
+/**
+ * driver adapter to drawer with several bugs.
+ */
+public class SolidLineDrawerAdapter implements Job2dDriver {
```

Comment:
The name is better but it lacks a driver aspect.

Confusing import. Direct coupling with DrawerFeature class is unnecessary.